static/scss/product-details.scss

```diff
@@ -0,0 +1,260 @@
+.container {
+  max-width: 1260px;
+  padding: 0 15px;
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+.hero-block {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
+  line-height: 30px;
+  color: $black;
+  padding: 20px 0;
+
+  @media (min-width: 768px) {
+    font-size: 24px;
+    line-height: 36px;
+  }
+
+  @media (min-width: 1024px) {
+    padding: 38px 0;
+  }
+
+  .content-row {
+    @media (min-width: 1024px) {
+      display: flex;
+      flex-flow: row wrap;
+      margin: 0 -15px;
+    }
+
+    .content-col {
+      padding: 0 0 20px;
+      @media (min-width: 768px) {
+        padding: 0 0 30px;
+      }
+      @media (min-width: 1024px) {
+        width: calc(100% - 455px);
+        padding: 0 15px;
+        box-sizing: border-box;
+      }
+
+      &.video {
+        padding-bottom: 0;
+        @media (min-width: 1024px) {
+          width: 455px;
+          padding: 0 15px;
+          flex-shrink: 0;
+        }
+      }
+    }
+  }
+
+  .video-holder {
+    background: $sirocco;
+    margin-top: 10px;
+
+    img {
+      display: block;
+      width: 100%;
+    }
+  }
+
+  .text-info {
+    @media (min-width: 1024px) {
+      max-width: 625px;
+    }
+  }
+
+  h1 {
+    font-size: 42px;
+    line-height: 50px;
+    margin: 0 0 13px;
+    font-weight: 700;
+
+    @media (min-width: 768px) {
+      font-size: 48px;
+      line-height: 60px;
+    }
+  }
+
+  p {
+    margin: 0 0 28px;
+  }
+
+  .btn.btn-default {
```

Comment:
One more comment: wherever possible we should be looking for styles that can be reused in other parts of the app. The "Enroll Now" button in the design has the same styling as many other buttons in other pages. You would expect any website to have some consistent styling for commonly-used elements like buttons, headers, borders, etc. This selector will style one single button in the product detail page, but looking at the designs, we know that this look and feel will be needed for other buttons. We should move these styles to a css class that any other button element can use (`.action-btn` or something like that)